### PR TITLE
GO-2041 | JS-2979 Sync widget update on account creation - spaces

### PR DESCRIPTION
**Pull request by KirillSto**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description

This PR fixes built-in usecases addition flow by sending events on widget blocks creation synchronously.

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

https://linear.app/anytype/issue/JS-2979/the-first-block-with-space-info-in-the-sidebar-is-absent

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

File: core/block/create.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func (s *Service) CreateWorkspace(ctx context.Context, req *pb.RpcWorkspaceCreat
 		return "", fmt.Errorf("set details for space %s: %w", spc.Id(), err)
 	}
 
-	_, err = s.builtinObjectService.CreateObjectsForUseCase(ctx, spc.Id(), req.UseCase)
+	_, err = s.builtinObjectService.CreateObjectsForUseCase(nil, spc.Id(), req.UseCase)
```
Comment (KirillSto) [inline core/block/create.go:197]: Won't this change break anything?

File: util/builtinobjects/builtinobjects.go [hunk 2]
```diff
@@ -243,7 +244,7 @@ func (b *builtinObjects) inject(ctx context.Context, spaceID string, useCase pb.
 		}
 	}()
 
-	if err = b.importArchive(ctx, spaceID, path); err != nil {
+	if err = b.importArchive(context.Background(), spaceID, path); err != nil {
```
Comment (KirillSto) [inline util/builtinobjects/builtinobjects.go:247]: Is using other context.Context from what comes in ObejctImportUseCase safe here?

**Review by deff7 — APPROVED** [review]

File: core/block/create.go (hunk 1 above)
Comment (deff7) [reply core/block/create.go:197]: test it manually

File: util/builtinobjects/builtinobjects.go (hunk 2 above)
Comment (deff7) [reply util/builtinobjects/builtinobjects.go:248]: I think it's ok, did you test it?

**Pull request merged**